Framework/Kernel/inc/MantidKernel/DataService.h

```diff
@@ -287,7 +287,7 @@ template <typename T> class DLLExport DataService {
     lock.unlock();
     notificationCenter.postNotification(new PreDeleteNotification(name, data));
     data.reset(); // DataService now has no references to the object
-    g_log.information("Data Object '" + name + "' deleted from data service.");
+    g_log.debug("Data Object '" + name + "' deleted from data service.");
```

Comment:
While we are changing this to be consistent with the other methods. Can you also change the `.information` in `rename`?

Reply:
Done :+1: